packages/react-components/react-field/src/components/FieldMessage/FieldMessage.types.ts

```diff
@@ -0,0 +1,26 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+/**
+ * FieldMessage Slots
+ */
+export type FieldMessageSlots = {
+  root: NonNullable<Slot<'div'>>;
+
+  /**
+   * Icon displayed before the message text. The default icon depends on the validationState.
+   */
+  icon?: Slot<'span'>;
+};
+
+/**
+ * FieldMessage Props
+ */
+export type FieldMessageProps = ComponentProps<FieldMessageSlots> & {
+  validationState?: 'error' | 'success' | 'warning' | 'neutral';
```

Comment:
Consider 'info'.  Most logging systems use 'info' rather than 'neutral' so it might be more familiar.  It would also avoid confusing with using neutral colors.

Reply:
`info` doesn't really make sense as a validation _state_ IMO. `none` is perhaps the best alternative. I can update it.

I'm adding this state is because of the change in default for validation state to `error` when there's a validation message. There needs to be a way to pick the styling of gray text with no icon (previous default).

Reply:
Another option for this is `validationState="custom"`, which is what it's called in the design spec. Not sure which is better between that and `validationState="none"`

Reply:
I ended up going with `none` in the PR that replaces this one: #26523